tools/firtool/firtool.cpp

```diff
@@ -773,6 +779,15 @@ static LogicalResult processBuffer(
         firrtl::createMergeConnectionsPass(
             !disableAggressiveMergeConnections.getValue()));
 
+  if (stripMuxPragmas) {
+    if (addMuxPragmas) {
+      emitError(module->getLoc())
+          << "--strip-mux-pragmas and --add-mux-pragmas are conflicting.";
+      return failure();
```

Comment:
nit: `return emitError` is a little more compact (and can drop the braces), Diagnostics auto-convert to LogicalResult conveniently.

Reply:
Consider moving this check as early as is easy/reasonable, to reject invalid invocations early (instead of mid-pipeline).

Reply:
Yep, moved to just after `runFirtool`